### ISSUE TITLE
[BUG] - IIs this library not working anymore?

### ISSUE THREAD

**Issue by rehovicova** [davidteather/TikTok-Api#953]
After trying countless of examples and workarounds, not getting a single result back, I am wondering, does this api even work? And if so, how do you make any example to work?

**Comment by Alex-Laparra** [comment]
No, it's been nonfunctional for months now.

**Comment by jovemadulto** [comment]
> No, it's been nonfunctional for months now.

I've managed to use it a few days ago. It seems like something has changed since then (maybe today?) as [tiktok-scraper](https://github.com/drawrowfly/tiktok-scraper) isn't working anymore too.

**Comment by popkarov** [comment]
It stopped working for me, around a month ago as well. Kept getting EmptyResponseExceptions, after a month where it worked fine.

**Comment by Tracer-Shield** [comment]
It stopped working for me today, keeps giving the same error on anything I try to do

**Comment by sjgiorgi** [comment]
Also experiencing this.

**Comment by Mishul-G** [comment]
Not working for me either. Anyone know of any alternatives?

**Comment by DeeFrancois** [comment]
It's currently working for me again but only if you import your own cookies. So just log in to TikTok (I used Chrome), and then use an extension to export all of the cookies to one file (I use Cookie-Editor). 

I was only able to test likes, user uploads, byHashtag, and getting video bytes. They all worked, and I never got any errors aside from the cookies expiring. Downloading videos hangs sometimes and the pull count isn't always accurate, but it does work well. Did multiple high-count pulls (anywhere from 200 to 2000 TikToks) without any issues. Hopefully this will last for a while.

Here's what my code looks like: 
```
with open('cookies.json') as f:
  cookies = json.load(f)

cookies_kv = {}

for cookie in cookies:
  cookies_kv[cookie['name']] = cookie['value']

cookies = cookies_kv

def get_cookies(**kwargs):
  return cookies

api = TikTokApi()
api._get_cookies = get_cookies

user_liked_list=list(api.user(username=self.username).liked(count=200))
```
Figured it out based on a comment from an Issue that I can't remember unfortunately.
